src/test/java/reactor/netty/transport/ProxyProviderTest.java

```diff
@@ -105,4 +114,12 @@ private ProxyProvider createHeaderProxy(InetSocketAddress address, Consumer<Http
 		                    .build();
 	}
 
+	private ProxyProvider createConnectTimeoutProxy(long connectTimeoutMillis) {
+		return ProxyProvider.builder()
+												.type(ProxyProvider.Proxy.SOCKS5)
+												.address(ADDRESS_1)
+												.connectTimeoutMillis(connectTimeoutMillis)
+												.build();
+	}
+
```

Comment:
please add tests to validate the documented default, the behavior when `connectTimeoutMillis` is set to `0` and when it is set to `-1`

Reply:
I did. 
thanks.